config/initializers/s3.rb

```diff
@@ -0,0 +1,8 @@
+CarrierWave.configure do |config|
+  config.fog_credentials = {
+      provider: 'AWS',
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.